var/spack/repos/builtin/packages/touchdetector/package.py

```diff
@@ -38,16 +38,20 @@ class Touchdetector(CMakePackage):
 
     variant('openmp', default=False, description='Enables OpenMP support')
 
-    depends_on('boost@1.50:')
     depends_on('cmake', type='build')
+    depends_on('boost@1.50:')
+    depends_on('hdf5@1.8:')
     depends_on('hpctools~openmp', when='~openmp')
     depends_on('hpctools+openmp', when='+openmp')
-    depends_on('hdf5@1.8:')
-    depends_on('libxml2')
+    depends_on('morphio@2.0.8:', when='@4.5:')
+    depends_on('mvdtool@1.5.1:', when='@4.5:')
```

Comment:
minor : there is no `4.5` version

Reply:
Yes, I know :)

This is for the future!